website/docs/language/backend/azurerm.mdx

```diff
@@ -596,8 +600,12 @@ When authenticating using a Service Principal with a Client Secret - the followi
 
 * `client_id` - (Optional) The Client ID of the Service Principal. This can also be sourced from the `ARM_CLIENT_ID` environment variable.
 
+* `client_id_file_path` (Optional) The path to a file containing the Client ID which should be used. This can also be sourced from the `ARM_CLIENT_ID_FILE_PATH` Environment Variable.
```

Comment:
```suggestion
- `client_id_file_path`: (Optional) Specifies the path to a file containing the client ID. Terraform presents the file client ID when authenticating with Azure. This can also be sourced from the `ARM_CLIENT_ID_FILE_PATH` Environment Variable.
```
Again, not sure if I'm describing the process correctly, but we should use active voice to make this information clear.